setup.cfg

```diff
@@ -0,0 +1,64 @@
+[metadata]
+name = ought-ice
+author = Ought
+author_email = team@ought.org
+license = MIT
+description = Interactive Composition Explorer
+url = https://github.com/oughtinc/ice
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Programming Language :: Python :: 3.10
+    License :: OSI Approved :: MIT License
+
+[options]
+packages = ice
+install_requires =
+    ruamel.yaml
+    Faker
+    defopt
+    fastapi
+    httpx
+    merge-args
+    nest-asyncio
+    nltk
+    numerizer==0.2.1
```

Comment:
Ah, I remember that, I think it's just because it isn't used by many people so Justin thought it was a bit more likely that an update would break us. That's less reasonable now that ice is becoming a proper library, so let's remove the constraint